### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by traversng**
| Before (opacity hack) | After (no hack) |
| --- | --- |
| image | image|

### CODE REVIEW

**Comment by traversng** [comment]
@siwan-cb I will address page container padding and max-width of layouts in a follow up PR

File: src/components/assets/Badge.tsx [hunk 1]
```diff
@@ -1,24 +1,108 @@
...
+    <path
+      d="M13 17.5L15.2857 19L19 13"
+      stroke="white"
+      strokeWidth="1.5"
+      strokeLinecap="square"
+    />
+  </svg>
+);
+
+const IncompleteSvg = () => (
```
Comment (siwan-cb) [inline src/components/assets/Badge.tsx:39]: use `memo`

File: src/components/assets/Badge.tsx [hunk 2]
```diff
@@ -1,24 +1,108 @@
...
+import { useMemo } from 'react';
+import Text from '@/components/Text';
 
 type Props = {
   imageUrl: URL | string;
   isCompleted: boolean;
   badge: BadgeType;
 };
+
+const CompletedSvg = () => (
```
Comment (siwan-cb) [inline src/components/assets/Badge.tsx:14]: use `memo`

File: src/components/assets/Badge.tsx [hunk 3]
```diff
@@ -1,24 +1,108 @@
...
+      d="M15 8.57735C15.6188 8.22008 16.3812 8.22008 17 8.57735L21.9282 11.4226C22.547 11.7799 22.9282 12.4402 22.9282 13.1547V18.8453C22.9282 19.5598 22.547 20.2201 21.9282 20.5774L17 23.4226C16.3812 23.7799 15.6188 23.7799 15 23.4226L10.0718 20.5774C9.45299 20.2201 9.0718 19.5598 9.0718 18.8453V13.1547C9.0718 12.4402 9.45299 11.7799 10.0718 11.4226L15 8.57735Z"
+      fill="black"
+    />
+    <path d="M13 13L19 19" stroke="white" strokeWidth="1.5" />
+    <path d="M13 19L19 13" stroke="white" strokeWidth="1.5" />
+  </svg>
+);
+
 export function Badge({ imageUrl, isCompleted, badge }: Props) {
+  const CompletedSvgMemo = useMemo(() => <CompletedSvg />, []);
```
Comment (siwan-cb) [inline src/components/assets/Badge.tsx:61]: better to use `memo`

**Comment by siwan-cb** [comment]
Opacity is a "hack" that I used. I think they need to be greyed out to match design.

**Pull request merged**